database/dbServices/shinx.api.js

```diff
@@ -55,15 +55,11 @@ export async function changeAutoFeed(id, mode) {
     return shinx.setAutoFeed(mode);
 };
 
-export async function addExperience(id, experience) {
-    let shinx = await getShinx(id, ['user_id', 'experience']);
-    const res = await shinx.addExperienceAndLevelUp(experience);
-    if (res.pre != res.post) {
-        if (hasPassedLevel(res.pre, res.post, 5)) await addEventTrophy(id, 'Bronze Trophy');
-        if (hasPassedLevel(res.pre, res.post, 15)) await addEventTrophy(id, 'Silver Trophy');
-        if (hasPassedLevel(res.pre, res.post, 30)) await addEventTrophy(id, 'Gold Trophy');
-        if (hasPassedLevel(res.pre, res.post, 50)) await addEventTrophy(id, 'Shiny Charm');
-    };
+export async function addBattleRewards(id, level) {
+    if (level > 5) await this.addEventTrophy(id, 'Bronze Trophy');
+    if (level > 15) await this.addEventTrophy(id, 'Silver Trophy');
+    if (level > 30) await this.addEventTrophy(id, 'Gold Trophy');
+    if (level > 50) await this.addEventTrophy(id, 'Shiny Charm');
```

Comment:
Will the other calls of `addExperience()` fail now that you deleted the function here or will they automatically use the `shinx.model.js` one? https://github.com/Glazelf/NinigiBot/blob/a21c43f77497fc8f1ba20ce7fc0cd7d557e208d6/database/dbObjects/models/userdata/shinx.model.js#L75
Is this accounted for?

Also, should `addBattleRewards()` not be called inside `addExperience()` so battle rewards are automatically given when experience is gained from any source, instead of needing to call `addBattleRewards()` every time exp is added?